app/models/project.rb

```diff
@@ -1,4 +1,20 @@
 class Project <ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def count_contestants
+    contestants.count
+  end
+
+  def average_years_exp
```

Comment:
No need to do all of this work. Let AR and SQL take care of this for you! Look into using aggregate functions like `average`